lib/rules/aria-allowed-attr.json

```diff
@@ -7,6 +7,6 @@
 		"help": "Elements must only use allowed ARIA attributes"
 	},
 	"all": [],
-	"any": ["aria-allowed-attr"],
+	"any": ["aria-allowed-attr", "aria-unsupported-attr"],
```

Comment:
What's the reasoning behind adding this to `aria-allowed-attr` instead of to `aria-valid-attr`? I haven't quite figured out which place I think is best, so I'd like to know your reasoning for it.

Reply:
It's a really fine line. I went for "allowed" because technically these attributes are valid, but we're recommending they not be used because of support purposes. Both `aria-allowed-attr` and `aria-valid-attr` use the same commons method to validate, except `aria-allowed-attr` has some more logic to check the role. The new unsupported check could go in either rule, really.

Reply:
I still believe that the correct rule for this behavior is `aria-allowed-attr` rather than `aria-valid-attr`, so we can get the correct messaging. I'm going to move forward with the change to the matches function so we can keep this rolling.